gatsby-config.js

```diff
@@ -3,16 +3,14 @@
 require('dotenv').config();
 
 module.exports = {
+  flags: { DEV_SSR: process.env.GATSBY_DEV_SSR || false },
   siteMetadata: {
-    siteTitle: 'Pixel Point Gatsby Starter', // <title>
+    siteTitle: 'Pixel Point Gatsby Starter',
     siteDescription: 'Site Description',
-    // pathPrefix: "",
-    siteImage: '/images/sample-image.png',
+    siteImage: '/images/social-preview.jpg',
     siteLanguage: 'en',
-    siteUrl: process.env.GATSBY_DEFAULT_SITE_URL || 'http://localhost:8000',
-    /* author */
-    authorName: 'pixel point',
-    authorTwitterAccount: '@',
+    siteUrl: process.env.GATSBY_DEFAULT_SITE_URL,
```

Comment:
Let's keep the default value here, or as an alternative we can have something like this:
```
const config = {
  GATSBY_DEFAULT_SITE_URL:  process.env.GATSBY_DEFAULT_SITE_URL || 'http://localhost:8000'
}
```
The same you can do with other variables.